WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1692,7 +1692,7 @@ - (void)uploadMedia:(Media *)media trackingId:(NSString *)mediaUniqueId
             [self.editorView replaceLocalVideoWithID:mediaUniqueId
                                       forRemoteVideo:media.remoteURL
                                         remotePoster:media.thumbnailLocalURL
-                                          videoPress:media.shortcode];
+                                          videoPress:media.videopressGUID];
```

Comment:
You might not have control over this but the parameter `videoPress` does not provide enough context. The other parameters are questionable. I know its outside the scope of this PR but it would provide clarity if it were something like:

```objc
- (void)replaceLocalVideoWithID:(NSString *)uniqueID
              forRemoteVideoURL:(NSString *)videoURL
                remotePosterURL:(NSString *)posterURL
                   videoPressID:(NSString *)videoPressID;
```

Something for @bummytime maybe? Not sure who "owns" that.